src/scripts/components/structures/header/MainNavigation.js

```diff
@@ -106,7 +98,7 @@ class MainNavigation extends Component<Props, Object> {
 
           {!this.props.isWalletSecured ? (
             <NavItem>
-              <NavLinkPurple
+              <ButtonColor
```

Comment:
The name `ButtonColor` seems confusing. That should be a color variable, I would think. What about `<ColoredButton />`?

Reply:
Could we do those changes with **[this](https://github.com/Paratii-Video/paratii-portal/pull/730#discussion_r196427367)** ?